### PR TITLE
GEODE-6070: Fix ShutdownCommandOverHttpDUnitTest flakiness

### PR DESCRIPTION
**Pull request by kirklund**
Use IgnoredException for DistributedSystemDisconnectedException.

### CODE REVIEW

File: geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandOverHttpDUnitTest.java [hunk 1]
```diff
@@ -96,48 +93,61 @@ public void setup() throws Exception {
...
     server1.invoke(() -> verifyNotConnected(SERVER_LAUNCHER.get().getCache()));
     server2.invoke(() -> verifyNotConnected(SERVER_LAUNCHER.get().getCache()));
     locator.invoke(() -> verifyNotConnected(LOCATOR_LAUNCHER.get().getCache()));
   }
 
   private void verifyNotConnected(Cache cache) {
-    await().untilAsserted(() -> assertThat(cache.getDistributedSystem().isConnected()).isFalse());
+    await().untilAsserted(() -> {
+      assertThat(cache.getDistributedSystem().isConnected())
+          .as("cache.getDistributedSystem().isConnected()")
```
Comment (jchen21) [inline geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandOverHttpDUnitTest.java:139]: Is it preferable to use `withFailMessage` than `as` for this case?

Comment (kirklund) [reply geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandOverHttpDUnitTest.java:139]: I use `as` to indicate an alias for the subject of the assertion, and `withFailMessage` as a full replacement of the failure message.

`distributed system isConnected` is probably a better example of English for an `as` in this case. It's kind of interesting to compare the output when using `as` vs `withFailMessage`.

Using `as`:
```
org.junit.ComparisonFailure: [cache.getDistributedSystem().isConnected()] 
Expected :false
Actual   :true
 <Click to see difference>


	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at org.apache.geode.test.awaitility.WithFailMessageTest.foo(WithFailMessageTest.java:79)
```
Using `withFailMessage`:
java.lang.AssertionError: cache.getDistributedSystem().isConnected()

	at org.apache.geode.test.awaitility.WithFailMessageTest.foo(WithFailMessageTest.java:79)
```
So I only use `withFailMessage` if I want to actually suppress and replace the native assertion failure message.

Comment (kirklund) [reply geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandOverHttpDUnitTest.java:139]: And this is the same assertion failure without `as` or `withFailMessage`:
```
org.junit.ComparisonFailure: 
Expected :false
Actual   :true
 <Click to see difference>


	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at org.apache.geode.test.awaitility.WithFailMessageTest.foo(WithFailMessageTest.java:78)
```

Comment (jchen21) [reply geode-web/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ShutdownCommandOverHttpDUnitTest.java:139]: Thank you for the explanation and examples!

**Pull request merged**